components/oracles-actions.tsx

```diff
@@ -79,10 +80,16 @@ function OraclesActions(): JSX.Element {
   }
 
   function onSuccess() {
+    setTokenAmount(0);
```

Comment:
We don't need to change the amount to 0, just the changeBalance update is enough